src/App.js

```diff
@@ -18,16 +19,19 @@ const Wrapper = styled.div`
 
 function App() {
   return (
-    <ThemeProvider theme={mainTheme}>
-      <BrowserRouter>
-        <Switch>
-          <Wrapper>
-            <Route exact path="/" component={Home} />
-          </Wrapper>
-          <Route exact path="/home" component={Home} />
-        </Switch>
-      </BrowserRouter>
-    </ThemeProvider>
+    <Provider>
```

Comment:
"Provider" é meio genérico, seria legal achar um nome mais específico pra esse cara.
```suggestion
    <Provider>
```